<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-D1O2GJ92" failures="1" tests="5" name="testRunners.TestNGCucuRunner" time="116.231" errors="0" timestamp="08 Jun 2020 13:44:11 GMT" skipped="0">
  <testcase classname="testRunners.TestNGCucuRunner" name="runScenario" time="22.129"/>
  <testcase classname="testRunners.TestNGCucuRunner" name="runScenario" time="22.065"/>
  <testcase classname="testRunners.TestNGCucuRunner" name="runScenario" time="24.930">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at stepDefinations.StepDefination01.user_should_get_something_message(StepDefination01.java:63)
at âœ½.user should get "Success - Check box is checked" message(file:///C:/Users/Gagana%20T/CucumberWithSelenium/CucumberTest02/SeleniumWithCucu/src/test/java/features/CheckBoxFeature.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunners.TestNGCucuRunner" name="runScenario" time="22.304"/>
  <testcase classname="testRunners.TestNGCucuRunner" name="runScenario" time="24.803"/>
</testsuite> <!-- testRunners.TestNGCucuRunner -->
